pkg/shell/superuser.jsx

```diff
@@ -180,6 +191,33 @@ const UnlockDialog = ({ proxy, host }) => {
             <Button variant="secondary" className='btn-cancel' onClick={cancel}>
                 {_("Close")}
             </Button>);
+    } else if (methods) {
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-17536-20220810-105234-1508028d-fedora-36-devel//Coverage/lcov/github-pr.diff.gcov.html)